isaaclab_arena/tasks/observations/observations.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2025, The Isaac Lab Arena Project Developers (https://github.com/isaac-sim/IsaacLab-Arena/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+
+from isaaclab.assets import RigidObject
+from isaaclab.envs import ManagerBasedRLEnv
+from isaaclab.managers import SceneEntityCfg
+from isaaclab.utils.math import subtract_frame_transforms
+
+
+def object_position_in_world_frame(
+    env: ManagerBasedRLEnv,
+    asset_cfg: SceneEntityCfg = SceneEntityCfg("object"),
+) -> torch.Tensor:
+    """Observation the position of the object in the world frame."""
+    object = env.scene[asset_cfg.name]
+    return object.data.root_pos_w
+
+
+def object_position_in_robot_root_frame(
+    env: ManagerBasedRLEnv,
+    robot_cfg: SceneEntityCfg = SceneEntityCfg("robot"),
+    object_cfg: SceneEntityCfg = SceneEntityCfg("object"),
+) -> torch.Tensor:
```

Comment:
I would say that this deserves a more generic name.

This get the position of object A in the frame of object B no? That's a very general function, there's nothing in here that's specific to a robot and a manipulated object.

Maybe we could call this `object_position_in_frame` and rename the variables accordingly?